src/main/java/io/netty/incubator/codec/quic/NoValidationQuicTokenHandler.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.incubator.codec.quic;
+
+import java.net.InetSocketAddress;
+
+import io.netty.buffer.ByteBuf;
+
+/**
+ * A {@link QuicTokenHandler} which disable token validation
+ */
+public final class NoValidationQuicTokenHandler implements QuicTokenHandler {
```

Comment:
I think we need this to disable server send retry packet which is needed for 0-RTT.

Reply:
So if 0-RTT is used there can be no validation ?

Reply:
Ideally we should have some validation on server side whether 0RTT is used or not, but this requires 'Initial packet' carries the a valid token which we does not support for now. So I think this can be a simple optional choice for 0RTT test. But still not sure if we can keep it in our code.

Reply:
I think in this case we should move this class to the test sources

Reply:
Done